package.json

```diff
@@ -18,7 +18,7 @@
     "start:visual:storybook": "(adb reverse tcp:7007 tcp:7007 || true) && start-storybook",
     "start:types": "tsc --w",
     "start:unit": "npm run test -- --watch",
-    "test": "./node_modules/jest/bin/jest.js ./src",
+    "test": "node ./node_modules/jest/bin/jest.js ./src",
```

Comment:
Just for my understanding—does Windows not just resolve binaries as you would expect?

```suggestion
    "test": "jest ./src",
```

Reply:
No this is due to https://github.com/expo/expo/issues/9129